src/main/kotlin/app/revanced/patches/idaustria/fingerprints/RootDetectionFingerprint.kt

```diff
@@ -0,0 +1,13 @@
+package app.revanced.patches.idaustria.fingerprints
+
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import org.jf.dexlib2.AccessFlags
+
+object RootDetectionFingerprint : MethodFingerprint(
+    "V",
+    access = AccessFlags.PUBLIC.value,
+    customFingerprint = { methodDef ->
+        methodDef.definingClass.endsWith("/DeviceIntegrityCheck;")
+                && (methodDef.name == "checkIntegrity" || methodDef.name == "checkIntegrityForceCheck")
```

Comment:
This check is redundant.

Reply:
Which check exactly? The method name check?